projects/ngrx.io/content/examples/store/src/app/my-counter/my-counter.component.ts

```diff
@@ -10,7 +10,7 @@ import { increment, decrement, reset } from '../counter.actions';
 export class MyCounterComponent {
   count$: Observable<number>;
 
-  constructor(private store: Store<{ count: number }>) {
+  constructor(private store: Store) {
```

Comment:
For this case, it's actually "required".
Because the example doesn't use a selector, but simply plucks the `count` state, the generic is required to have the extra type safety. E.g. if you would do `store.select('coun')`, TypeScript would catch this and throw an error.